lib/cylc/conditional_simplifier.py

```diff
@@ -19,6 +19,7 @@
 import re
 import ast
 import copy
+import sys
```

Comment:
Why is sys being imported? Perhaps I'm being blind but I can't see it being used (new glasses here :grin:)